### ISSUE TITLE
Update jcabi-parent to 0.49.4 (solves xcop issue)

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#1047]
Let's update to new jcabi version (0.49.4) because this new version solves xcop issue on windows.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by llorllale** [comment]
@fabriciofx done in c1f043aa891c75a00aaec41cfedc5eedb1f17f84

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1047` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Comment by fabriciofx** [comment]
@llorllale Just for curiosity, what are the points you consider to put an issue in scope?

**Comment by llorllale** [comment]
@fabriciofx you made me stop to think :)

See, the point was to focus on releasing `1.0`. Pretty soon, all you guys will have > 1024 points or whatever and will be kicked out of this sandbox project. :(

But I *have* been including tasks not necessary for `1.0`, which means I haven't been consistent. I will rectify this.

**Comment by fabriciofx** [comment]
@llorllale I've asked it because I've opened many issues and corrected the bug in `jcabi-parent` and I haven't get any points for them. I made this contribution because I like these projects and love contribute with open source, but if I can do it and receive some points at Zerocracy too, it's even better, right? 😄 

Besides you made me think about it too: I think interesting have a possibility to someone with more than 1024 points and was kicked of a sandbox project, returned if he/she want to. WDYT?
